### ISSUE TITLE
Private Message view

### ISSUE THREAD

**Issue by Nutomic** [LemmyNet/lemmy-ui#376]
Currently the only wayt to view private messages is view notifications. This is pretty awkward because it mixes messages from different users (and even posts/comments if you dont select the "messages" view explicitly).

So it would be good to have a separate page which only shows a user selection, and private messages exchanged with that user. Depending how it is implemented, this could even turn Lemmy into somewhat of a chat app (we already have live updates).

**Comment by dessalines** [comment]
I spose it would be possible to create a Chat like view that would look more like Element or something. I'd probably leave the one in notifications alone, because notifs can still be several types.

That also begs the question of whether a chat like view is also possible for posts, as one user suggested a few weeks back. IE on the left tab could be users, or posts you've contributed to.

**Comment by Nutomic** [comment]
Yes notifications should stay like they are, this would be a separate, additional feature. And I think that we should keep it as simple as possible for the initial version. After thats released we can think about adjustments or additions.

**Comment by kekkoudesu** [comment]
The main feature I want is the ability to see the messages I've sent somebody.

**Comment by dessalines** [comment]
You can definitely see messages you've sent, but we're reworking everything for 1.0 notifications.
